defaults/main.yml

```diff
@@ -1,21 +1,26 @@
 ---
-kea_apt_repo_version: 2-3
-kea_apt_key_fingerprint: 95B6C7ABDAEE9F164891B962DA05D46B7BABA24A
+kea_apt_repo_version: 2-4
+kea_apt_key_fingerprint: 4AD10B6C98E09742E530EC4F0D9D9A1439E23DB9
```

Comment:
I realise now we forgot to set the default value for `kea_apt_key_fingerprint` in `meta/argument_specs.yml`, would you kindly add this and set it to `4AD10B6C98E09742E530EC4F0D9D9A1439E23DB9`.